api/test/unit/models/index.test.js

```diff
@@ -1,12 +1,11 @@
 'use strict';
 
-//var expect = require('expect.js');
 var expect = require("chai").expect;
-var logger = require('../../lib/log/logger');
+var logger = require('../../../lib/log/logger');
 
 describe('Model Initialization', function () {
   it('returns the Policy model', function () {
-    var models = require('../../lib/models');
+    var models = require('../../../lib/models')();
     logger.log('Policy model loaded successfully');
     expect(models.policy_json).to.be.ok;
```

Comment:
This only tests what you expect, not what may actually be there.
Don't we also expect sequelize which also means there should be only 2 items in the map?

I could add more models and never change this test which would always pass.